src/validation.cpp

```diff
@@ -1488,15 +1488,18 @@ bool CheckInputs(const CTransaction& tx, CValidationState &state, const CCoinsVi
                     pvChecks->push_back(CScriptCheck());
                     check.swap(pvChecks->back());
                 } else if (!check()) {
-                    if (flags & STANDARD_NOT_MANDATORY_VERIFY_FLAGS) {
+                    const bool hasNonMandatoryFlags = (flags & STANDARD_NOT_MANDATORY_VERIFY_FLAGS) != 0;
+                    const bool doesNotHaveDip0020Opcodes = (flags & SCRIPT_ENABLE_DIP0020_OPCODES) == 0;
```

Comment:
I don't like negative booleans like this.. I'd like it to be adjusted to `hasDip0020Opcodes`

Then change to fit format `has_DIP0020_opcodes`